pkg/reconciler/domainmapping/reconciler.go

```diff
@@ -121,16 +121,9 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, dm *v1alpha1.DomainMappi
 		return err
 	}
 
-	// Set HTTPOption via config-network.
-	var httpOption netv1alpha1.HTTPOption
-	switch config.FromContext(ctx).Network.HTTPProtocol {
-	case networkingpkg.HTTPEnabled:
-		httpOption = netv1alpha1.HTTPOptionEnabled
-	case networkingpkg.HTTPRedirected:
-		httpOption = netv1alpha1.HTTPOptionRedirected
-	// This will be deprecated soon
-	case networkingpkg.HTTPDisabled:
-		httpOption = ""
+	httpOption, err := httpOption(ctx, dm.GetAnnotations())
```

Comment:
bit of a nit, but all the other blocks have a comment, would be nice to have one here too

Reply:
add comment